web/themes/custom/contribtracker/cypress/fixtures/urls_endpoint.csv

```diff
@@ -0,0 +1,12 @@
+url,title
```

Comment:
We don't need a CSV parser complicating our dependency tree for such a simple CSV file. In fact, we can just write a simple JS file returning a config array or a JSON file. Something like this:

```json
{
  "Home": "/",
  "CodeContributions", "/code-contributions",
  // and so on...
}
```

Reply:
Yes, let's not use CSV. We can use a JSON file or just hardcode the array in the spec file itself.

Reply:
Will go with JSON file

Reply:
We have switched to JSON, right? Why still keep the CSV file?